Scalar.Common/Platforms/Linux/LinuxPlatform.cs

```diff
@@ -19,6 +19,25 @@ public partial class LinuxPlatform : POSIXPlatform
         // TODO(Linux): determine installation location and upgrader path
         private const string UpgradeProtectedDataDirectory = "/usr/local/scalar_upgrader";
 
+        // TODO(Linux): We should ideally consider any colon-separated paths
+        // in $XDG_CONFIG_DIRS and $XDG_DATA_DIRS, as well as their defaults
+        // (i.e., /etc/xdg and /usr/local/share:/usr/share).
+        // We should also ideally create any missing directories using a 0700
+        // permission mode via a native wrapper for mkdir(2) instead of
+        // relying on our caller's use of Directory.CreateDirectory().
+        private static readonly EnvironmentVariableBasePath[] EnvironmentVariableBaseCachePaths = new[] {
+            new EnvironmentVariableBasePath("XDG_CACHE_HOME", "scalar"),
+            new EnvironmentVariableBasePath("HOME", Path.Combine(".cache", "scalar")),
```

Comment:
Repeating my comment from #422: Should `"HOME"` be in `ScalarConstants`? Add the string `"XDG_CACHE_HOME"` to that list, too.